### PR TITLE
16.0 mig account_template_active

### PR DESCRIPTION
**Pull request by legalsylvain**
quite simple migration 

    - use new web_ribbon widget to display archived items on form views
    - change inheritance from account to account_usability regarding views that disappeared in V16

- depends on : https://github.com/OCA/account-financial-tools/pull/1477

### CODE REVIEW

**Comment by rafaelbn** [comment]
/ocabot migration account_template_active

@legalsylvain could you please rebase?

**Comment by legalsylvain** [comment]
@StefanRijnhart, @jcdrubay : could you take a look on this trivial PR ? 
thanks !

File: account_template_active/README.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+* Go to Configuration > Templates
```
Comment (rafaelbn) [inline account_template_active/README.rst:77]: - Go to Accounting -> Configuración -> Templates

**Review by rafaelbn — APPROVED** [review]
Tested in runboat 👍

**Review by StefanRijnhart — APPROVED** [review]
Thanks! Please fix adding the top Accounting menu in the readme as Raphael suggests.

File: account_template_active/README.rst (hunk 1 above)
Comment (legalsylvain) [reply account_template_active/README.rst:77]: Done in readme/USAGE.rst. (the mentioned file is autogenerated)

**Comment by legalsylvain** [comment]
https://github.com/OCA/oca-github-bot/issues/226 is definitively annoying.

**Review by rafaelbn — APPROVED** [review]
/ocabot merge patch

**Pull request merged**